api/components/textfield.schema.json

```diff
@@ -12,57 +12,123 @@
             ]
         },
         "value": {
-            "description": "the value displayed in the element",
+            "description": "The value displayed inside the Textfield",
             "type": "string"
         },
-        "label": {
-            "description": "the text displayed aside the textfield",
-            "type": "string"
-        },
-        "hintText": {
-            "description": "the text displayed inside the textfield",
-            "type": "string"
+        "autocorrect": {
+            "description": "Whether to enable the autocorrection",
+            "type": "boolean"
         },
-        "description": {
-            "description": "the text displayed under the textfield",
-            "type": "string"
+        "autofillHints": {
+            "description": "The type of this text input to provide autofill hints.",
+            "type": "array",
+            "items": {
+                "$ref": "defs/autofillHints.schema.json"
+            }
         },
-        "obscure": {
-            "description": "Whether the input text should be hidden or not",
+        "autofocus": {
+            "description": "Whether this Textfield should be focused initially.",
             "type": "boolean"
         },
-        "disabled": {
-            "description": "Whether the input should be disabled or not",
+        "buildCounter": {
+            "description": "Callback that generates a custom counter widget.",
+            "$ref": "defs/listener.schema.json"
+        },
+        "controller": {
```

Comment:
Remove controller as it can break some things on the textfield, this will need a specific issue.